wordpress-mobile/gutenberg-mobile
# Add initial VideoPress test for Android

This PR introduces two key changes:

1. **Initial test for VideoPress block:** A new test has been added to ensure the successful insertion of the VideoPress block into the editor. This test checks the visibility of the VideoPress block and verifies the expected HTML output. For now, this test only runs on Android, iOS support will be introduced in a separate PR.

2. **Update to testMatch pattern:** The testMatch pattern in the project's Jest configuration has been adjusted to include subdirectories under the 'test' directory, thus enabling us to create a nested directory for VideoPress. 

To test: 

* Run `npm run test src/test/videopress/edit.js` from the project's directory.
* Ensure all tests pass as expected on this PR.

<hr />

PR submission checklist:
